pyds8k/resources/ds8k/v1/cs/pprcs.py

```diff
@@ -0,0 +1,57 @@
+##############################################################################
+# Copyright 2019 IBM Corp.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##############################################################################
+
+"""
+advanced PPRC interface.
+"""
+
+from ..common.base import Base, ReadOnlyManager
+from ..common.types import DS8K_CS_PPRC
+from ..volumes import Volume, VolumeManager
+from ..systems import System, SystemManager
+
+
+class CSPPRC(Base):
```

Comment:
cspprc is not an accurate resource name, it is a resource under another base endpoint (api/v1/cs), consider changing the name

Reply:
1. Resource for /v1/pprc has been named PPRC and this resource should be different with it. 
2. This resource shouldn't be PPRCS because it should similar with other resources style, like resource Volume for /v1/volumes.
Because of these, I named this resource CSPPRC. Do you have any advice about its name?